ckan_cloud_operator/providers/ckan/instance/manager.py

```diff
@@ -284,6 +312,32 @@ def set_name(instance_id, instance_name, dry_run=False):
         kubectl.apply(resource)
 
 
+def set_storage(instance_id, instance_name, dry_run=False):
+    from ckan_cloud_operator.providers.storage.manager import get_provider
+
+    resource = crds_manager.get(INSTANCE_NAME_CRD_SINGULAR, name=instance_name, required=False)
+
+    cluster_provider_id = cluster_manager.get_provider_id()
+    if cluster_provider_id == 'gcloud':
+        provider_id = 'gcloud'
+    elif cluster_provider_id == 'aws':
+        provider_id = 's3'
+    else:
+        logs.warning(f'No storage module found for cluster provider {cluster_provider_id}')
+        return
+
+    storage_provider = get_provider(default=None, provider_id=provider_id)
+    bucket = storage_provider.create_bucket(instance_id, exists_ok=True, dry_run=dry_run)
+    resource['spec']['bucket'] = {
```

Comment:
let's be more verbose here - ckanStorageBucket instead of just 'bucket'

Reply:
done